### PR TITLE
Add more notification types to listen to and reload accounts on

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers

### CODE REVIEW

**Comment by cjb** [comment]
@keybase/react-hackers bump

File: shared/actions/notifications.js [hunk 1]
```diff
@@ -47,11 +47,13 @@ const setupEngineListeners = () => {
     NotificationsGen.createReceivedBadgeState({badgeState})
   )
 
-  getEngine().setIncomingActionCreators('stellar.1.notify.paymentNotification', ({accountID}) =>
-    WalletsGen.createRefreshPayments({accountID})
-  )
+  getEngine().setIncomingActionCreators('stellar.1.notify.paymentNotification', refreshPayments)
+  getEngine().setIncomingActionCreators('stellar.1.notify.paymentStatusNotification', refreshPayments)
+  getEngine().setIncomingActionCreators('stellar.1.notify.requestStatusNotification', refreshPayments)
```
Comment (buoyad) [inline shared/actions/notifications.js]: I don't think we need this, this notification doesn't affect payments AIUI

**Comment by cjb** [comment]
Thanks @buoyad.  @patrickxb, any reason to reload payment details on receiving a `requestStatusNotification`?

**Comment by patrickxb** [comment]
Nope.

**Pull request merged**